01-data-structures/03-linked-lists/linked_list.rb

```diff
@@ -4,27 +4,120 @@ class LinkedList
   attr_accessor :head
   attr_accessor :tail
 
+  def initialize
+    @head = nil
+    @tail = nil
+  end
+
+  def find_index(number)
```

Comment:
why did you create these? the point of a linkedlist is that it's not indexed

Reply:
I created those extra methods because the answer section wanted me to run tests comparing accessing specific indices using array vs. linked list

Reply:
Ohhhhhhhhhhhh